consensus/istanbul/backend.go

```diff
@@ -97,9 +99,67 @@ type Backend interface {
 	// ParentBlockValidators returns the validator set of the given proposal's parent block
 	ParentBlockValidators(proposal Proposal) ValidatorSet
 
-	// RefreshValPeers will connect with all the validators in the validator connection set and disconnect validator peers that are not in the set
-	RefreshValPeers() error
-
 	// Authorize injects a private key into the consensus engine.
 	Authorize(address common.Address, publicKey *ecdsa.PublicKey, decryptFn DecryptFn, signFn SignerFn, signBLSFn BLSSignerFn)
 }
+
+// BackendForProxy provides the Istanbul backend application specific functions for Istanbul proxy
```

Comment:
Simmilar to above, I would suggest moving this interface into the `proxy` package instead of renaming it.

Reply:
Done.